pkg/validation/internal/csv.go

```diff
@@ -55,6 +55,10 @@ func parseCSVNameFormat(name string) error {
 	if violations := k8svalidation.IsDNS1123Subdomain(name); len(violations) != 0 {
 		return fmt.Errorf("%q is invalid:\n%s", name, violations)
 	}
+	// Give CSV name is used as label value, it should be validated
+	if errs := k8svalidation.IsValidLabelValue(name); errs != nil {
+		return fmt.Errorf("%q is invalid: %s", name, strings.Join(errs[:], ","))
+	}
```

Comment:
```suggestion
	var errStrs []string
	errStrs = append(errStrs, k8svalidation.IsDNS1123Subdomain(name)...)
	// Give CSV name is used as label value, it should be validated
	errStrs = append(errStrs, k8svalidation.IsValidLabelValue(name)...)
	
	if len(errStrs) > 0 {
		return fmt.Errorf("%q is invalid:\n%s", name, strings.Join(errStrs, ","))
	}
	return nil
```

Reply:
You could also just return a `return utilerrors.NewAggregate(errs)` as that package is already present as a vendored dependency, instead of explicitly checking the length of any list of errors and aggregating an error string.

Reply:
The returned type from those validation functions is `[]string` not `[]error`, so you cannot use `NewAggregate()` directly.

Reply:
Ah, gotcha yeah I should've checked the return type there - good catch. I guess there's no immediate way to wrap a list of strings into a single error to make that type work, so disregard my suggestion.